ansible_collections/arista/avd/requirements.txt

```diff
@@ -0,0 +1,9 @@
+netaddr==0.7.19
+Jinja2==2.11.3
```

Comment:
jinja, PyYAML and others are requirement of ansible itself too, although it is alright here, but I would avoid pinning them to a specific version

Reply:
@anshulbehl - appreciate the feedback, since our collection heavily relies on jinja2, we like the have specific control over its version. Are there any issues in doing so?

Reply:
The issue is with the multiple collections inside a single execution environment and each specifying their own version, we can see conflicts and have problem building the containers at that point, if there is lower limit you can specify that would be better(if that is an option), we will be solving that by specifying upper constraints.
Hope this helps!!

Reply:
@carlbuchmann I see this is merged, but as per my previous comment please avoid pinning the requirements to a specific version if it is not needed, and specify a range instead.